### ISSUE TITLE
Suggestion: More form of malicious compliance

### ISSUE THREAD

**Issue by Goncalerta** [username-is-required/reddark#129]
Some subs are using other forms of malicious compliance that degrades the quality of content while still listening to the will of the users. For example r/interestingasfuck is reinterpreting its sub to be about anything the users find interesting; r/programmerhumor is implementing weekly votes to change rules, some subs are implementing Touch-grass tuesdays, etc.

It would be great to try and track as much as possible all the different forms of protest, to show that the community is still united even though they are being forced to open.

**Comment by username-is-required** [comment]
i agree that it would be great to do, and if i thought it was feasible it’s something i’d love to implement. however, while keeping track of specific forms of malicious compliance (such as the john oliver movement) can be done with the help of a [script](https://github.com/username-is-required/reddark-subinfo-update-checker), effectively maintaining a list of all subs taking part in malicious compliance more ***generally*** would - i feel - be virtually impossible.

i went into some more detail as to why over at https://github.com/username-is-required/reddark-subinfo/issues/1#issuecomment-1596225343: tl;dr: if another form of unique (trackable) protest gains popularity, i can adjust my script to also allow me to track it; but it’s virtually impossible (unfortunately) to track all subreddits engaged in general malicious compliance, because of not being able to be alerted to possible changes to when subs start/change/stop maliciously complying.

regarding touch-grass tuesdays, i think i would lean against adding a specific “touch-grass tuesdays” label to my fork, as reddark will already notice & display if a sub has gone private/restricted for that day.

(if i haven’t answered your question fully or you have another then please do ask, i’m just conscious that i might have inadvertently skipped over something you wrote or not answered something fully as i’m quite tired at the moment!)

**Comment by Goncalerta** [comment]
Yeah, that makes total sense.
Maybe one possibility would be to have some place to manually submit a `(subreddit, protest_stickied_post)` pair, to be reviewed and then added to the site. This would require assuming that the protest would be in effect while the post is stickied, and also that there would be a way to see if the post is still stickied. This would be the most general way I can think of, however possibly involves way too manual intervention.

The problem of touch-grass tuesdays is that, currently, while the sub is not in closed/restricted mode, it appears as completely public, which makes it seem like it's no longer protesting. Basically my idea would be to be able to track as many subreddits as possible in order to show how significant the amount of protests still is even after many are coming back open. But I totally understand that this might be completely unfeasible.


For what is worth, here are some protesting subs
[Welcome back! What's next?](https://www.reddit.com/r/ProgrammerHumor/comments/14dqb6f/welcome_back_whats_next/)
[r/interestingasfuck will be reopening Monday June 19th with rule changes.](https://www.reddit.com/r/interestingasfuck/comments/14c0vcf/rinterestingasfuck_will_be_reopening_monday_june/)
[Opening r/eurovision, touching grass, and going forward](https://www.reddit.com/r/eurovision/comments/14dddes/opening_reurovision_touching_grass_and_going/)
[“Touch-Grass Tuesday” - Why this subreddit is locked down](https://www.reddit.com/r/TheOwlHouse/comments/14e1zr9/touchgrass_tuesday_why_this_subreddit_is_locked/)
[r/bonehurtingjuice STANDS WITH THE ADMINS and the principles of the Communist Party of China. We will not shut down a revolutionary platform ✊](https://www.reddit.com/r/bonehurtingjuice/comments/147xe86/rbonehurtingjuice_stands_with_the_admins_and_the/)
[Rule Changes for r/WellThatsSucks - All About Vacuum Cleaners! The Story of Mr. Wells](https://www.reddit.com/r/Wellthatsucks/comments/14c8x0z/rule_changes_for_rwellthatssucks_all_about_vacuum/)
[📢 Important Announcement: Embracing Change on Our Christian Minecraft Server! 🌟](https://www.reddit.com/r/horny/comments/14an8n5/important_announcement_embracing_change_on_our/)
[A Dashing Change to r/iPhone Rules: Let's Celebrate the Charm of Tim Cook!](https://www.reddit.com/r/iphone/comments/14cb2za/a_dashing_change_to_riphone_rules_lets_celebrate/)
[New direction for the sub](https://www.reddit.com/r/Nofans/comments/14crgpo/new_direction_for_the_sub/)
[r/TheyKnew is open again! With a twist. Read pinned comment for the new requirement to post on the subreddit.](https://www.reddit.com/r/theyknew/comments/14boze6/rtheyknew_is_open_again_with_a_twist_read_pinned/)

Seems difficult, but maybe searching keywords like "Rule changes" or "touch grass" in stickied posts could help.

**Comment by robertafreese** [comment]
Hard to believe no one a MaliciousCompliance is recording these and the lack is itself a little bit suspicious.
 
I have been keeping a list of these for my own amusement and would share and monitor if this project and the related project is interested in the data. I do not know how much longer I will be able to put daily time into monitoring this list as most of my activity is mobile on 3rd party apps. So it would be weekends only most likely.
